### PR TITLE
Refactor team initialize to be consistent

### PR DESCRIPTION
**Pull request by wally-yawn**
Refactor to use a row object rather than individual variables in team initialize method.

### CODE REVIEW

File: spec/team_spec.rb
Comment (Crosswolfv1) [inline spec/team_spec.rb:1]: You have a binding pry on line 11 please remove

Comment (wally-yawn) [reply spec/team_spec.rb:1]: done, thanks

**Review by Crosswolfv1 — APPROVED** [review]
Verified changes are correct to the desired formatting, issue with commenetd code is resolved Approved

**Review by Montana-Pfeifer — COMMENTED** [review]
changes look good for refactored code

**Pull request merged**